akka-docs/src/main/paradox/general/addressing.md

```diff
@@ -154,15 +154,15 @@ other actors are found; its name is `"/"`. The next level consists of the
 following:
 
  * `"/user"` is the guardian actor for all user-created top-level actors;
-actors created using `ActorSystem.actorOf` are found below this one.
+actors created using @apidoc[ActorSystem.actorOf](ActorRefFactory) {scala="#actorOf(props:akka.actor.Props):akka.actor.ActorRef" java="#actorOf(akka.actor.Props)"} are found below this one.
  * `"/system"` is the guardian actor for all system-created top-level actors,
 e.g. logging listeners or actors automatically deployed by configuration at
 the start of the actor system.
  * `"/deadLetters"` is the dead letter actor, which is where all messages sent to
 stopped or non-existing actors are re-routed (on a best-effort basis: messages
 may be lost even within the local JVM).
  * `"/temp"` is the guardian for all short-lived system-created actors, e.g.
-those which are used in the implementation of `ActorRef.ask`.
+those which are used in the implementation of @apidoc[ActorRef.ask](ExplicitAskSupport) {scala="#ask(actorRef:akka.actor.ActorRef):akka.pattern.ExplicitlyAskableActorRef" java="#ask(akka.actor.ActorRef)"}.
```

Comment:
I'm not sure about these links. Is it right?

Reply:
I think `akka.pattern.AskableActorRef#ask` is what you end up using if you import the implicit ask for  Scala through `akka.patterns.ask` and call `.ask` (for classic actors), for Java there is no `ActorRef.ask` it is always explicitly `akka.pattern.Patterns#ask`

No Scaladoc on `akka.pattern.AskableActorRef#ask` though.